action.go

```diff
@@ -280,6 +191,173 @@ func (c *cmdAction) String() string {
 	return "[" + strings.Join(ss, " ") + "]"
 }
 
+// CmdConfig allows creating Actions for commands using custom logic for
```

Comment:
```
CmdConfig is used to create redis command Actions with particular settings. All
fields are optional, all methods are thread-safe.

The global Cmd and FlatCmd functions are shortcuts for using an empty
CmdConfig{}.
```

No need to document ActionProperties on the global level, that can be done on the field.

Reply:
Done